src/Organ.java

```diff
@@ -39,6 +45,10 @@ public void changeResistance(double delta) {
         if (resistance > 1) { resistance = 1; }
     }
 
+    public void runCombat() {
+
+    }
+
     public abstract void applyCollapsePenalties(ImmuneSystem system);
     public abstract void applyDamage(Pathogen cell);
```

Comment:
Provide documentation for what applyDamage refers to. Is it the organ applying damage to the pathogen or vice versa?